assets/js/view/part/city.rb

```diff
@@ -87,7 +103,7 @@ def render_part
 
         children = []
 
-        children << render_box(slots.size) if (2..3).include?(slots.size)
+        children << render_box(slots.size) if (2..6).include?(slots.size)
```

Comment:
this should be

slot.size.between(2, 6)

Reply:
Right now getting: `undefined method `between' for 1`

Reply:
oh fuck, ?, fixing